plugins/source/aws/client/spec.go

```diff
@@ -83,3 +84,9 @@ func validateOUs(ous []string) error {
 	}
 	return nil
 }
+
+func (s *Spec) SetDefaults() {
+	if s.InitializationConcurrency <= 0 {
+		s.InitializationConcurrency = 1
```

Comment:
```suggestion
		s.InitializationConcurrency = 4
```
Maybe default to an 'harmless' value such as 4?

Reply:
Isn't the most harmless value `1`?

Reply:
There is no gain when using it with `1` which is the same as using the old code.